conf/test_full.config

```diff
@@ -5,7 +5,7 @@
     Defines input files and everything required to run a full size pipeline test.
 
     Use as follows:
-        nextflow run icgc-argo-workflows/prealnqc -profile test_full,<docker/singularity> --outdir <OUTDIR>
+        nextflow run icgc-argo-workflows/prealnqc -profile test_full,<standard/singularity> --outdir <OUTDIR>
```

Comment:
keep as `docker` instead of `standard`?

Reply:
@edsu7 that's a walk around for running jobs in RDPC, as the `standard` profile is used by default when no profile is specified: https://www.nextflow.io/docs/latest/config.html#config-profiles
Currently rdpc does not support specify profile yet.